src/index.js

```diff
@@ -1,45 +1,52 @@
 import './style.css';
+import { Task, getData, saveData } from './task.js';
+import UI from './ui.js';
+import display from './display.js';
+import Status from './status.js';
 
-const dataToDoList = [
-  {
-    description: 'Jumping',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Running',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Eating',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Drinking',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Sleeping',
-    completed: true,
-    index: 4,
-  },
-];
-function populateList(tasks) {
-  const containerList = document.querySelector('.todo');
-  let listHTML = '';
-  tasks.forEach((task) => {
-    listHTML += `
-      <li class="list-to-do">
-        <input type="checkbox" class="to-do">
-        ${task.description}
-        <i class="fa-solid fa-trash"></i>
-      </li>
-      <hr class="line">
-    `;
-  });
-  containerList.innerHTML = listHTML;
+let tasksList;
+if (getData() === null) {
+  tasksList = [];
+} else {
+  tasksList = getData();
 }
-populateList(dataToDoList);
+
+const addTask = (newTask) => {
+  let index;
+  if (getData() === null) {
+    index = 1;
+  } else {
+    tasksList = getData();
+    index = tasksList.length + 1;
+  }
+  const task = new Task(newTask, false, index);
+  tasksList.push(task);
+  saveData(tasksList);
+  display(tasksList);
+};
```

Comment:
- [ ] Please your project should contain methods related to the status updates (completed: true/false) I see that you are passing the index instead of false this is causing an error since the index which is a number is a truthy value as you see when the user press delete all completed it remove all the items in the list.
![image](https://user-images.githubusercontent.com/74744787/216791069-467739f1-5843-464a-95f6-1164dece1bba.png)
![image](https://user-images.githubusercontent.com/74744787/216791096-d0db4652-3f24-4a8b-860c-a062de5ed3b5.png)
![image](https://user-images.githubusercontent.com/74744787/216791137-a7354712-149d-408f-bb8e-aebf76646b56.png)